scripts/get-changelog.js

```diff
@@ -17,11 +17,11 @@
 // This code will generate changelog entries
 
 const { readFile } = require('fs');
+const { promisify } = require('util');
 // eslint-disable-next-line import/no-extraneous-dependencies
 const fetch = require('isomorphic-fetch');
```

Comment:
Can we remove this dep altogether? It's a polyfill for browsers, but we're using this script only with node.js

Reply:
Sure, thanks @anshgoyalevil